#include "stdafx.h"
#include "vlc.h"

vlc::vlc(QWidget *parent)
	: QMainWindow(parent)
{
	ui.setupUi(this);
	ui.label->setText("14023");
	ui.label_2->setText("01-25-2017");
	ui.label_3->setText("0023");
	ui.label_4->setText("K1DG");
	ui.label_5->setText("05");
	ui.label_5->setText("9Y4VT");
	ui.label_6->setText("09");
}
